run.py

```diff
@@ -196,11 +205,16 @@ def get_all_the_issues():
         metrics='ga:totalEvents',
         dimensions='ga:eventLabel',
         sort='-ga:totalEvents',
-        #fields='rows, columnHeaders'
         filters='ga:eventCategory==Civic Issues;ga:eventLabel=@github.com').execute()
 
-    total_issues=results["rows"]
-    return total_issues
+    issues = []
+    for row in results["rows"]:
+        issue = {
+            "url" : row[0],
+            "clicks" : row[1]
```

Comment:
We were talking about correlating time of a click to an action, should I add a function to get a "last_clicked", plus the google analytics timestamp of that time?